### PR TITLE
Elaborate more on watchdog dump action

### PR DESCRIPTION
**Pull request by ahadas**
Related to https://bugzilla.redhat.com/show_bug.cgi?id=1665449

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @ahadas 

This pull request needs review by: @stoobie @dcdacosta  @emarcusRH

### CODE REVIEW

File: source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_High_Availability_settings_explained.adoc [hunk 1]
```diff
@@ -53,7 +53,7 @@ The following options are available:
 
 * *poweroff* - The virtual machine is immediately shut down.
 
-* *dump* - A dump is performed and the virtual machine is paused.
+* *dump* - A dump is performed and the virtual machine is paused. The guest's memory is dumped by libvirt, hence neither kdump nor pvpanic is required. The dump file is created in the directory that is configured by auto_dump_path in file /etc/libvirt/qemu.conf on the host.
```
Comment (dcdacosta) [inline source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_High_Availability_settings_explained.adoc:56]: Suggest changing to the following for grammatical and formatting accuracy:
The guest's memory is dumped by libvirt, therefore, neither 'kdump' nor 'pvpanic' is required. The dump file is created in the directory that is configured by auto_dump_path in the /etc/libvirt/qemu.conf file on the host.

**Review by dcdacosta — APPROVED** [review]
A few minor changes; otherwise, LGTM!

File: source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_High_Availability_settings_explained.adoc (hunk 1 above)
Comment (ahadas) [reply source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_High_Availability_settings_explained.adoc]: done, thanks

**Review by sandrobonazzola — APPROVED** [review]

**Comment by sandrobonazzola** [comment]
Merging based on approval by @dcdacosta and on rephrasing added by @ahadas addressing the comments from the review.

**Pull request merged**